scripts/components/app.simulation.component.ts

```diff
@@ -0,0 +1,95 @@
+import { Component, OnInit, NgZone, ElementRef } from '@angular/core';
+import { WorkItemsService } from '../services/work-items.service';
+
+import { WorkItem } from '../Model/WorkItem';
+
+import { select } from 'd3-selection';
+import { scaleLinear } from 'd3-scale';
+
+@Component({
+    selector:'simulation',
+    templateUrl:'./app.simulation.component.html'
+})
+export class SimulationComponent implements OnInit{
+
+    el: HTMLElement;
+
+    public CompletedWorkItems : WorkItem[];
+
+    public CompletedWorkItemsCount : number;
+
+    public InProgressWorkItems: WorkItem[];
+    
+    public InProgressWorkItemsCount : number;
+
+    public NumberOfIterations: number;
+
+    constructor(private workItemsService : WorkItemsService, 
+                private zone : NgZone,
+                private elementRef: ElementRef){
```

Comment:
nitpicking: `private` here creates a private field holding the ElementRef. It looks like you're only interested in the `nativeElement` property since I can't see `this.elementRef` being used anywhere.